geom/alg_rotating_calipers.go

```diff
@@ -78,17 +97,18 @@ func findBestMBR(seq Sequence) rotatedRectangle {
 		}
 		lhs.update(seq, i)
 
-		area := rhs.proj.Sub(lhs.proj).Cross(far.proj)
-		if area < bestArea {
-			bestArea = area
-			rect = rotatedRectangle{
-				origin: seq.GetXY(i).Add(lhs.proj),
-				span1:  rhs.proj.Sub(lhs.proj),
-				span2:  far.proj,
-			}
+		canadidateRect := rotatedRectangle{
+			origin: seq.GetXY(i).Add(lhs.proj),
+			span1:  rhs.proj.Sub(lhs.proj),
+			span2:  far.proj,
+		}
+		candidateMetric := metric(canadidateRect)
+		if i == 0 || candidateMetric < bestMetric {
```

Comment:
It looks like we're getting the minimum metric, what do you think about renaming `best` -> `min` (e.g. `findMinMBR`, etc.) to be more explicit?